### PR TITLE
UI2 exchange scene

### PR DESCRIPTION
**Pull request by andreyvEze**
confetti:
https://user-images.githubusercontent.com/22767467/116308623-65447500-a7b0-11eb-8c77-332787be1b53.mov

iPhone 12:
12 exch
12 exch selected
12 quote

iPad:
iPad exch
iPad exch selected
iPad exch success
iPad exch quote

iPod:
touch quote
touch congr
touch exch
touch exch selected

### CODE REVIEW

**Comment by jmcranska** [comment]
Screenshot #2: we need to remove the "Next" button, as the current flow of the app allows the done button the keyboard to move to the next scene. Also, would like all components to match the very thin/faint border style used in Screenshot #3

File: src/components/themed/ThemedButtons.js [hunk 1]
```diff
@@ -143,7 +143,9 @@ export function RightChevronButton(props: { text: string, onPress: () => void, p
   return (
     <TouchableOpacity onPress={onPress}>
       <View style={[spacingStyles({ marginRem: [0, 1.25, 0, 0], paddingRem }, theme), styles.rightChevronContainer]}>
-        <EdgeText style={styles.rightChevronText}>{text}</EdgeText>
+        <EdgeText ellipsizeMode="middle" style={styles.rightChevronText}>
```
Comment (andreyvEze) [inline src/components/themed/ThemedButtons.js:146]: waiting the answer from Paul

File: src/components/themed/Fade.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+// @flow
+
+import * as React from 'react'
+import { type AnimatedValue, Animated } from 'react-native'
```
Comment (andreyvEze) [inline src/components/themed/Fade.js:4]: use reanimated

Comment (andreyvEze) [reply src/components/themed/Fade.js:4]: wait fro Julia to respond about the button

**Comment by jmcranska** [comment]
For the button on the final congratulations and scene, the button should just say "Done"

**Comment by paullinator** [comment]
Please generate all screenshots with the keyboard showing in the simulator. Control-K should do the job.

**Comment by paullinator** [comment]
vertical spacing also looks a bit excessive. On smaller screens a lot of the content at the bottom will not be visible. Try reducing total space between each vertical component to 1 REM.

**Comment by paullinator** [comment]
Change logic for the confetti cannon to fire with gradually decreasing frequency. Use the following algorithm and save state to the local context disklet

1. Show confetti the first, second, sixth, and 12th swap
2. Randomly show confetti one out of eight times.

**Comment by andreyvEze** [comment]
@paullinator Spacings reduced, screenshots with clipboard changed

**Comment by thehobbit85** [comment]
replaced by https://github.com/EdgeApp/edge-react-gui/pull/2597

**Pull request closed without merging**